lang/idris2/Portfile

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           makefile 1.0
+
+name                idris2
+github.setup        idris-lang Idris2 0.7.0 v
+categories          lang
+license             BSD
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+description         Purely functional programming language with first class types
```

Comment:
Just FYI, this is a terrible description. What makes Idris special is that it allows real programming with dependent types, so you can prove your programs are correct in the type system.

Reply:
I just copied it, of course, not authored. I will change it according to your suggestion.

Reply:
Done.